docs/faq.rst

```diff
@@ -306,3 +306,45 @@ Access full logs
 
 If you want to access the full logs you need to write ``-q`` and ``-v`` as
 individual tox arguments and avoid combining them into a single one.
+
+Running within a Docker container
+---------------------------------
+
+If you want to run tox within a Docker container you can use `31z4/tox <https://hub.docker.com/r/31z4/tox>`_.
+This Docker image neatly packages tox along with common build dependencies (e.g., ``make``, ``gcc``, etc) and currently
+`active CPython versions <https://devguide.python.org/versions/#status-of-python-versions>`_. See more details in
+its `GitHub repository <https://github.com/31z4/tox-docker>`_.
+
+The recommended way of using the image is to mount the directory that contains your tox configuration files and your
+code as a volume. Assuming your project is within the current directory of the host, use the following command to run
+tox without any flags:
+
+.. code-block:: shell
+
+    docker run -v `pwd`:/home/tox/tests -it --rm 31z4/tox
+
+Because an entry point of the image is ``tox``, you can easily pass subcommands and flags:
+
+.. code-block:: shell
+
+    docker run -v `pwd`:/home/tox/tests -it --rm 31z4/tox run-parallel -e black,py311
+
+Note, that the image is configured with a working directory at ``/home/tox/tests``.
+
+If you want to install additional Python versions/implementations or Ubuntu packages you can create a derivative image.
+Just make sure you switch the user to ``root`` when needed and switch back to ``tox`` afterwards:
+
+.. code-block:: Dockerfile
+
+    FROM 31z4/tox
```

Comment:
Could we create a tox-dev or tox org instead to host under? You can be one of it's owner's 😊

Reply:
Yes I think it makes sense. Although, if we aim to make the image "official" maybe it's better to work towards this right now. This way we can save some maintenance efforts. Because we don't have to deprecate `tox-dev/tox` in favour of official `tox`. And also users don't have to migrate from the deprecated image to the official one.

Some benefits of an official image that I'd like to highlight:
* Automated multi-arch builds.
* Automated periodic re-builds when a base image is updated.
* Better structured docs and build artefacts (e.g.. https://github.com/docker-library/repo-info/tree/master/repos/zookeeper).

Reply:
Yeah let's make it so. What do you need from me to get there?

Reply:
I guess nothing at this point. I'll make some usability changes to the image to conform with https://github.com/docker-library/official-images#consistency. And then will work on PRs to https://github.com/docker-library/official-images and https://github.com/docker-library/docs.